nservicebus/sagas/index.md

```diff
@@ -83,11 +56,44 @@ This interface tells NServiceBus that the saga not only handles `StartOrder`, bu
 partial: at-least-one
 
 
+### Dealing with out of order delivery
```

Comment:
@weralabaj I don't understand why this is nested under *A simple saga*

Reply:
it's not, it'd be under "Starting a saga" header, https://docs.particular.net/nservicebus/sagas/#starting-a-saga

Reply:
I thought it'd make sense, since that part concentrates on caveats of staring a saga.

Reply:
@weralabaj Sorry, that was due github not showing all text during the compare.

In my opinion the discarding behavior and out or order are mentioned too late. I think they are key elements to mention very early on. Maybe we need an introduction at the top? Stating the things like stateful, correlation, out of order and discarding?

Reply:
Hm... looking at the whole article I don't have a good idea of where to put it earlier, first we explain what saga is, and then move on to starting... So quite early in fact. Although I agree the article is getting a bit long and maybe that's the problem?

Not sure, open for edits/specific suggestions.